themis/languages/java/installation-android.md

```diff
@@ -35,6 +35,33 @@ If you experience difficulties with jcenter repository, try using maven instead:
 
 Once JavaThemis is installed, you can [try out examples on your machine](../examples/).
 
+
+## Configuring ProGuard / R8 rules
+
+Sometimes when working with Android projects in `Release` configuration, you might get runtime error, like:
```

Comment:
This section is about a particular runtime error, not about dealing with runtime errors in general. I'd rephrase the sentence a bit to suggest that what follows is an example. Also, clauses usually don't need commas.

```suggestion
Sometimes when working with Android projects in `Release` configuration you might get the following runtime error:
```